cmd/registry/registryStart.go

```diff
@@ -21,9 +21,79 @@ THE SOFTWARE.
 */
 package registry
 
-import "github.com/spf13/cobra"
+import (
+	log "github.com/sirupsen/logrus"
+
+	"github.com/rancher/k3d/v4/pkg/runtimes"
+	k3d "github.com/rancher/k3d/v4/pkg/types"
+
+	"github.com/rancher/k3d/v4/pkg/client"
+
+	"github.com/spf13/cobra"
+)
+
+type regStartFlags struct {
+	All bool
+}
 
 // NewCmdRegistryStart creates a new cobra command
 func NewCmdRegistryStart() *cobra.Command {
-	return &cobra.Command{}
+	flags := &regStartFlags{}
+
+	// create new command
+	cmd := &cobra.Command{
+		Use:   "start (NAME | --all)",
+		Short: "Start a registry/registries",
+		Long:  `Start a registry/registries`,
+		Args:  cobra.MaximumNArgs(1), // maximum one name accepted
```

Comment:
In L88 you're actually looping over the **list** of args, so this should not limit it to 1, right?